internal/security/config/command/proxy/adduser/command_test.go

```diff
@@ -52,24 +52,47 @@ func TestAddUserBadArg(t *testing.T) {
 
 // TestAddUserJWT tests functionality of adduser command using JWT option
 func TestAddUserJWT(t *testing.T) {
-	// Arrange
+
+	// Create a mock logger and grab the default configuration struct
 	lc := logger.MockLogger{}
 	config := &config.ConfigurationStruct{}
+
+	// Test variables
+	tokenType := "jwt"
+	jwt := "s0meJWT"
+	user := "someuser"
+	algorithm := "RS256"
+	publicKey := "testdata/rsa.pub"
+
+	// Create a mock server for handling the command requests
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		// Check to make sure the JWT authorization header exists
+		assert.NotNil(t, r.Header.Values("Authorization"))
+		require.Equal(t, "Bearer "+jwt, r.Header.Values("Authorization")[0])
```

Comment:
use constants for "Authorization" and "Bearer "

Reply:
This is a unit test.  Shouldn't use the same constant as the main code base if suggestion is implemented.